spec/features/practice_editor/checklist_spec.rb

```diff
@@ -1,115 +1,115 @@
 require 'rails_helper'
 
-describe 'Practice editor', type: :feature, js: true do
```

Comment:
we can remove this file right?

Reply:
Yeah - i'll remove it - checklist probably won't come back..  :)